test/unit/sidekick.test.js

```diff
@@ -545,16 +545,19 @@ describe('Test sidekick bookmarklet', () => {
 
   it('Publish plugin also purges dependencies', async () => {
     const actionHost = 'https://adobeioruntime.net';
-    const purgePath = '/en/topics/foo.html';
-    let purged = false;
+    const dependencies = [
+      '/en/topics/foo.html',
+      'bar.html?step=1',
```

Comment:
@davidnuescheler please check if the test reflects the intention of this change.